tools/diagnostic-information

```diff
@@ -164,15 +164,18 @@ system_logs() {
 ipm_logs() {
     verbose "ipm logs..."
     # FIXME: check for activemq and NGINX logs
+    mkdir -p "${WD}/${TMPDIR}/logs-karaf"
     if [ -n "${KARAF_LOG}" ] && [ -d "${KARAF_LOG}" ]
     then
         for logfile in $(ls "${KARAF_LOG}")
         do
             verbose "adding $logfile..."
-            cp -H -f "${KARAF_LOG}/${logfile}" "${WD}/${TMPDIR}/logs/${logfile}"
+            cp -H -f "${KARAF_LOG}/${logfile}" "${WD}/${TMPDIR}/logs-karaf/${logfile}"
```

Comment:
if making a separate dedicated dir, I'd be in favor of naming it "logs-ipm" or "logs-ipm2" since it can/will holds everything around the specifics of IPM2 (karaf, emc4j, activemq, ...), as implied by the function name (ipm-logs) ;)

Reply:
...and then different products would each have their own error.log piled one over another?

Reply:
Followed up with my initially suppressed gut feeling, to put `ipm2` in the dedicated karaf log dir name.

Reply:
The `stderr` point followed up in https://github.com/42ity/fty-core/pull/343